src/Store.js

```diff
@@ -46,6 +46,12 @@ class Store {
     this.dbname = address.path || ''
     this.events = new EventEmitter()
 
+    this.remoteHeadsPath = address + '/_remoteHeads'
+    this.localHeadsPath = address + '/_localHeads'
+    this.snapshotPath = address + '/snapshot'
+    this.queuePath = address + '/queue'
+    this.manifestPath = address + '/_manifest'
```

Comment:
I think we should move away from using string operations for paths and would be preferable to use `path.join(address, '_remoteHeads')` etc